funnel/models/proposal.py

```diff
@@ -8,6 +8,7 @@
 from .commentvote import CommentSpace, VoteSpace, SPACETYPE
 from coaster.utils import LabeledEnum
 from coaster.sqlalchemy import SqlSplitIdComparator
+from coaster.sqlalchemy.statemanager import StateManager
```

Comment:
Import `StateManager` from `coaster.sqlalchemy`. Add it to the previous line's imports.